returnn/torch/engine.py

```diff
@@ -1472,8 +1472,7 @@ def _print_process(
             info += ["%.3f sec/step" % step_duration]
         if start_elapsed is not None:
             info += ["elapsed %s" % hms(start_elapsed)]
-        if complete_frac is not None:
-            assert 1 >= complete_frac > 0, f"{step} step, {complete_frac} complete_frac"
+        if complete_frac is not None and 1 >= complete_frac > 0:
```

Comment:
I would leave the check in. Specifically, instead of `None`, we also allow `-1` as value to have this disabled. But for anything else, it should be valid, i.e. in the range (0,1].
```suggestion
        if complete_frac not in (-1, None):
            assert 1 >= complete_frac > 0, f"{step} step, {complete_frac} complete_frac"
```